api/PerformanceTiming.json

```diff
@@ -884,36 +941,76 @@
             },
             "safari_ios": {
               "version_added": false
+            },
+            "webview_android": {
+              "version_added": true
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": false
+            "deprecated": true
           }
         }
       },
       "toJSON": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceTiming/toJSON",
-          "support": {},
+          "support": {
+            "chrome": {
+              "version_added": "45"
```

Comment:
This was added in Chrome 44/Opera 32. Please change.

Reply:
@jpmedley since this has already been merged and Florian's going on holiday, the fastest fix here would be for you to open a PR and then I can review it and merge it. Thank you!